plugin/EasyGrep.vim

```diff
@@ -819,7 +819,9 @@ function! <sid>EchoGrepCommand()
     if s:GetGrepCommandName() == "grep"
         let shellCommand = substitute(grepCommand, "grep", &grepprg, "")
         call s:Echo("Shell command:         ".shellCommand)
-        let @* = shellCommand
```

Comment:
Actually this line should simply be deleted entirely. I think this was accidentally left in unintentionally when debugging something.